e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -799,6 +788,39 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
       horizontal: -100,
     });
   });
+
+  it("should not crash notebook when metric is used as an aggregation and breakout is applied (metabase#40553)", () => {
+    createMetric({
+      name: "Revenue",
+      description: "Sum of orders subtotal",
+      table_id: ORDERS_ID,
+      definition: {
+        "source-table": ORDERS_ID,
+        aggregation: [["sum", ["field", ORDERS.SUBTOTAL, null]]],
+      },
+    }).then(({ body }) => {
+      cy.wrap(body.id).as("metricId");
+    });
+
+    cy.get("@metricId").then(metricId => {
+      const questionDetails = {
+        query: {
+          "source-table": ORDERS_ID,
+          breakout: [["field", ORDERS.CREATED_AT, null]],
+          aggregation: [["metric", metricId]],
+        },
+      };
+      cy.createQuestion(questionDetails, { visitQuestion: true });
```

Comment:
```suggestion
      createQuestion(questionDetails, { visitQuestion: true });
```

Custom command `createQuestion` has been deprecated. Please use a helper instead.
![image](https://github.com/metabase/metabase/assets/31325167/c9b242a9-d058-401f-ac06-cb9af590d3c7)